src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -29,6 +29,7 @@
 using Microsoft.CodeAnalysis.Symbols;
 using Microsoft.DiaSymReader;
 using Roslyn.Utilities;
+using ReferenceEqualityComparer = Roslyn.Utilities.ReferenceEqualityComparer;
```

Comment:
Longer term should we possibly rename this to `RoslynReferenceEqualityComparer` to avoid this using everywhere? Or maybe just unify on the BCL version? @sharwell, @RikkiGibson

Reply:
BCL version looks like it would behave the same as ours https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Collections/Generic/ReferenceEqualityComparer.cs

Reply:
IReadOnlySet we would want to keep, though, I guess. Ours has way fewer members on it than the platform version.